apps/www/_blog/2021-04-02-supabase-pgbouncer.mdx

```diff
@@ -96,4 +96,4 @@ Try out connection pooling now with a new project in the [dashboard.](https://ap
 
 Eventually, we will expose more PgBouncer settings to the UI such as `Pool Size`. At the moment it is set to `15`.
 
-We are still working towards getting the latest version of Supabase Postgres to both the AWS and Digital Ocean marketplaces. Follow us on [Twitter](https://www.notion.so/PgBouncer-is-now-available-in-Supabase-345a74d402464670923cf404b714a354) to be informed once it's released!
+We are still working towards getting the latest version of Supabase Postgres to both the AWS and Digital Ocean marketplaces. Follow us on [Twitter](https://supabase.com) to be informed once it's released!
```

Comment:
This link should be: https://twitter.com/supabase

Reply:
fixed